src/labels/CustomLabels.labels

```diff
@@ -1907,6 +1907,14 @@
         <shortDescription>Migration section warning</shortDescription>
         <value>&lt;b&gt;WARNING: Run Migration modifies existing Recurring Donation data to match the Enhanced Recurring Donations model. Backup your database if you haven't already done so.&lt;/b&gt;</value>
     </labels>
+    <labels>
+        <fullName>RD2_EnablementPermissionRequired</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error displayed when user does not have System Admin permissions</shortDescription>
```

Comment:
@sam-knox , can you please review the new custom label?

Reply:
@RadSFDeveloper - Sure thing. I assume we're checking for Customize Application, Author Apex, and Modify All Data. Anything else?

Reply:
Only those permissions, the same three that they are checked for the NPSP Settings page access.

Reply:
@sam-knox , the custom label text has been updated. Thanks for the feedback.

Reply:
@ravensisco , this is the text displayed when a user does not have permission to run RD2 data migration (AC5 scenario).